src/util/mapResize.js

```diff
@@ -0,0 +1,33 @@
+// set the map menu to the correct size
```

Comment:
What is this file for?

Reply:
Whoops forgot to remove that. I ended up not using it.